pkg/apis/apps/v1beta1/register.go

```diff
@@ -57,6 +57,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&Scale{},
 		&StatefulSet{},
 		&StatefulSetList{},
+		&ControllerRevision{},
```

Comment:
The only issue with doing that is that StatefulSet and DaemonSet update will break if the API's aren't enabled by default.